<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThemeDialog</class>
 <widget class="QDialog" name="ThemeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Theme</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="renderLayout" stretch="0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="pageCombo">
         <item>
          <property name="text">
           <string>Main - VariWatt</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Main - Temp. Control</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Main - Power Bank</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Main - Misc.</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Menu</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Info, Dialogs</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Charge</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Screensaver</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="lv">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorBtn">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="btnLayout">
     <item>
      <widget class="QPushButton" name="exportBtn">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importBtn">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="readBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Read from device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="writeBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Write to device</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
